CLIENT-SPECIFICATION.md

```diff
@@ -173,29 +173,37 @@ If multiple versions of a page were found for different platforms, then a client
+Pages can be written in multiple languages. If a client has access to environment variables, it MUST use them derive the preferred user language as described in the next paragraphs. If not, then clients MUST make reasonable assumptions based on the information provided by the environment in which they operate (e.g. consulting `navigator.languages` in a browser, etc.).
 
-The [`LANG` environment variable](https://www.gnu.org/software/gettext/manual/html_node/Locale-Environment-Variables.html), if present, MUST be used to determine the language of pages to display.
+The [`LANG` environment variable](https://www.gnu.org/software/gettext/manual/html_node/Locale-Environment-Variables.html) specifies the user preferred locale (in the form `ll[_CC][.encoding]`). The [`LANGUAGE` environment variable](https://www.gnu.org/software/gettext/manual/html_node/The-LANGUAGE-variable.html) specifies a priority list of locales (in the form `l1:l2:...`) that can be used if the locale defined by `LANG` is not available. Both `LANG` and `LANGUAGE` may contain the values `C` or `POSIX`, which should be ignored.
 
-The [`LANGUAGE` environment variable](https://www.gnu.org/software/gettext/manual/html_node/The-LANGUAGE-variable.html) specifies a priority list of languages that a user wishes to read in. The `LANG` environment variable MUST be present for `LANGUAGE` to be used, and then `LANGUAGE` takes precendence over `LANG`. If `LANG` is set to a language not in `LANGUAGE`, then `LANG` should be appended to the end of the priority list. If a page is not available in the user's preferred language, then a client MUST respect the user's priority list defined in the `LANGUAGE` variable, and MAY choose to notify the user that a page in their chosen language couldn't be found (perhaps along with a link to the [translations section of the contributing guide](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#translations)).
+In order to determine the display language, a client MUST:
 
-Regardless of the language selected through the above environment variables, clients MUST always attempt to fallback to English if the page does not exist in the requested languages. In this case clients SHOULD tell the user that the page does not exist in their requested language if it was not English. If the client supports a command-line argument for language, the client MUST only attempt to show the page in that language (clients MAY notify the user that a page is available in other languages if present).
+1. Check the value of `LANG`.
```

Comment:
What about this?
```suggestion
1. Check the value of `LANG`. If not set, then skip to step 5.
```

Reply:
I think it's not what is intended, since `LANGUAGE` (step 2) can contain the information, too.

Reply:
Looking at https://github.com/tldr-pages/tldr/pull/4101#issuecomment-650645755, which demonstrates different values of `LANG` and `LANGUAGE` for `gettext`, it would seem that `LANG` must be set for `LANGUAGE` to have any effect.

Reply:
@sbrl @MasterOdin ops, you're right guys, thanks for spotting this.